lte/gateway/c/core/oai/test/n11/test_smf_service_client.cpp

```diff
@@ -75,8 +75,7 @@ TEST(test_create_sm_pdu_session_v4, create_sm_pdu_session_v4) {
       rat_req->mutable_gnode_endpoint()->end_ipv4_addr());
```

Comment:
Here i have the same question as below, why are use using the same IP as subscriber ip for gnode ?

Reply:
ACL has written this code and we'll fix it later on for all.

Reply:
Hey @mehul-jindal-wavelabs  this is just a matter of setting a different ip for each field so the test can catch the issue. In fact the issue this PR is fixing is partially caused for using the same ip. Should we have used a different ip for each field, the test would have failed.

Please, change the ips, use predefined values for each inputs so you can check the outputs.

Reply:
agree with your point, if both IPs are not the same.
the test case has different values.
UE PDU session IP: ipv4_addr("10.20.30.44")   
GNB IP address   : ("10.20.30.40")

Reply:
Thanks @ganeshg87 . Now it looks how it should be

Note the point is not to make this value different because the test output is different. The approach is: we need different inputs because in the function code those are independent parametrs.